src/services/api/features/package-actions/services/error-handle-service.ts

```diff
@@ -0,0 +1,5 @@
+export class APIError extends Error {
+  constructor(message: string) {
+    super(message);
+  }
+}
```

Comment:
I would add a directory `services/api/core` and dump `error-handle-service.ts` there.